stats/view/aggregation_data.go

```diff
@@ -89,18 +92,35 @@ func (a *SumData) equal(other AggregationData) bool {
 //
 // Most users won't directly access distribution data.
 type DistributionData struct {
-	Count           int64     // number of data points aggregated
-	Min             float64   // minimum value in the distribution
-	Max             float64   // max value in the distribution
-	Mean            float64   // mean of the distribution
-	SumOfSquaredDev float64   // sum of the squared deviation from the mean
-	CountPerBucket  []int64   // number of occurrences per bucket
-	bounds          []float64 // histogram distribution of the values
+	Count           int64   // number of data points aggregated
+	Min             float64 // minimum value in the distribution
+	Max             float64 // max value in the distribution
+	Mean            float64 // mean of the distribution
+	SumOfSquaredDev float64 // sum of the squared deviation from the mean
+	// Deprecated: Replaced by Count in Buckets.
```

Comment:
Perhaps let's put this field at the bottom and a blank space before it to draw attention to it like this:
```go
     bounds []float64 // histogram distribution of the values
     Buckets []Bucket
     
     // Deprecated: Replaced by Count in Buckets.
     CountPerBucket []int64 // Number of occurances per bucket
```

Reply:
ok, I think this makes sense especially given that most Exporters won't care about exemplars.